frontend/src/layout/lemonade/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,34 +1,47 @@
+import React, { useState } from 'react'
 import { useValues } from 'kea'
-import React from 'react'
 import { IconExpandMore } from '../../../lib/components/icons'
 import { Link } from '../../../lib/components/Link'
 import './Breadcrumbs.scss'
 import { Breadcrumb as IBreadcrumb, breadcrumbsLogic } from './breadcrumbsLogic'
 import { Tooltip } from '../../../lib/components/Tooltip'
 import clsx from 'clsx'
 import { Skeleton } from 'antd'
+import { Popup } from 'lib/components/Popup/Popup'
 
 function Breadcrumb({ breadcrumb }: { breadcrumb: IBreadcrumb }): JSX.Element {
+    const [popoverShown, setPopoverShown] = useState(false)
+
     let breadcrumbContent = (
-        <div className={clsx('Breadcrumbs__breadcrumb', breadcrumb.here && 'Breadcrumbs__breadcrumb--current')}>
+        <div
+            className={clsx(
+                'Breadcrumbs__breadcrumb',
+                breadcrumb.here && 'Breadcrumbs__breadcrumb--current',
+                (breadcrumb.path || breadcrumb.popup) && 'Breadcrumbs__breadcrumb--actionable'
+            )}
+            onClick={() => breadcrumb.popup && setPopoverShown(!popoverShown)}
+        >
             {breadcrumb.symbol}
             {breadcrumb.name}
         </div>
     )
+
     if (breadcrumb.path) {
         breadcrumbContent = <Link to={breadcrumb.path}>{breadcrumbContent}</Link>
     }
-    let { tooltip } = breadcrumb
-    if (!tooltip) {
-        if (breadcrumb.path) {
-            tooltip = `Go to ${breadcrumb.name}`
-        } else if (breadcrumb.here) {
-            tooltip = 'You are here'
-        }
+
+    if (breadcrumb.popup) {
+        return (
+            <Popup {...breadcrumb.popup} visible={popoverShown} onClickOutside={() => setPopoverShown(false)}>
+                {breadcrumbContent}
+            </Popup>
+        )
     }
-    if (tooltip) {
-        breadcrumbContent = <Tooltip title={tooltip}>{breadcrumbContent}</Tooltip>
+
+    if (breadcrumb.tooltip) {
+        return <Tooltip title={breadcrumb.tooltip}>{breadcrumbContent}</Tooltip>
     }
```

Comment:
Looks like this doesn't work anymore (outside of mostly having gone out of use)

Reply:
Removed